app/src/views/private/components/drawer-item.vue

```diff
@@ -151,7 +151,15 @@ const swapFormOrder = computed(() => {
 	return props.junctionFieldLocation === 'top';
 });
 
-const hasEdits = computed(() => !isEmpty(internalEdits.value));
+const hasEdits = computed(() => {
+	if (isEmpty(internalEdits.value)) return false;
+
+	if (props.junctionField && Object.keys.length === 1 && isEmpty(internalEdits.value[props.junctionField]))
```

Comment:
What does `Object.keys.length === 1` do?

Reply:
Always equal to true. 😅